client/data/site-verticals/use-site-verticals-featured.ts

```diff
@@ -2,23 +2,28 @@ import { useQuery, UseQueryResult, QueryKey } from 'react-query';
 import wpcom from 'calypso/lib/wp';
 import type { SiteVerticalsResponse } from './types';
 
-const useSiteVerticalsFeatured = (): UseQueryResult< SiteVerticalsResponse[] > => {
-	return useQuery( getCacheKey(), () => fetchSiteVerticalsFeatured(), {
+const useSiteVerticalsFeatured = ( {
+	locale = 'en',
+} ): UseQueryResult< SiteVerticalsResponse[] > => {
```

Comment:
This approach has a big issue: by introducing the new argument, all other callers need to be updated. We need to make it possible for the object argument to be undefined as well as the inner `locale` field.